controllers/machinepools.go

```diff
@@ -10,17 +10,28 @@ import (
 	arbv1 "github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/apis/controller/v1beta1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog"
-	"os"
 	"strings"
 )
 
-func scaleMachinePool(aw *arbv1.AppWrapper, userRequestedInstanceType string, replicas int) {
+type OCMService interface {
```

Comment:
Can we add this interface to a different file?

Reply:
Great suggestions, thanks so much for the feedback @asm582